src/ansible_navigator/_yaml.py

```diff
@@ -62,8 +74,15 @@ def represent_scalar(self, tag, value, style=None):
         return super().represent_scalar(tag, value, style)
 
 
-# from http://stackoverflow.com/a/15423007/115478
-def _is_multiline_string(value):
+def _is_multiline_string(value: str):
+    """Determine if a string is multiline.
+
+    Note:
+    from http://stackoverflow.com/a/15423007/115478
```

Comment:
We can use an RST admonition for this:
```suggestion
    .. note::

       Inspired by http://stackoverflow.com/a/15423007/115478.
```

Reply:
Or maybe it's better to be
```suggestion
    .. tip::

       Inspired by http://stackoverflow.com/a/15423007/115478.
```
(or "hint"?)

Reply:
got it